src/public/index.html

```diff
@@ -60,6 +60,18 @@ <h1 id="rpsStat">-</h1>
       <canvas id="rpsChart" width="200" height="100"></canvas>
     </div>
   </div>
+  <div class="container">
+    <div class="stats-column">
+      <h5>Status Codes</h5>
+      <h6 style="color: #75D701">2xx</h6>
+      <h6 style="color: #47b8e0">3xx</h6>
+      <h6 style="color: #ffc952">4xx</h6>
+      <h6 style="color: #E53A40">5xx</h6>
```

Comment:
why you are not adding 4 classes to the existing **style.css** file?

`/src/public/stylesheets/style.css`
```css
.status-code-2xx {
    color: #75D701;
}

.status-code-3xx {
    color: #47b8e0;
}

.status-code-4xx {
    color: #ffc952;
}

.status-code-5xx {
    color: #E53A40;
}
```

`/src/public/index.html`
```html
<h5>Status Codes</h5>
<h6 class="status-code-2xx">2xx</h6>
<h6 class="status-code-3xx">3xx</h6>
<h6 class="status-code-4xx">4xx</h6>
<h6 class="status-code-5xx">5xx</h6>
```